Packages/Netherlands3D/Windmills/Runtime/Scripts/Windmill.cs

```diff
@@ -0,0 +1,89 @@
+using System.Linq;
+using Netherlands3D.Events;
+using Netherlands3D.T3DPipeline;
+using UnityEngine;
+
+namespace Netherlands3D.Windmills
+{
+    public class Windmill : MonoBehaviour
+    {
+        private const string AXIS_HEIGHT_KEY = "asHoogte";
+        private const string ROTOR_DIAMETER_KEY = "rotorDiameter";
+        private const string STATUS_KEY = "status";
+
+        public float RotorDiameter { get; private set; }
+        public float AxisHeight { get; private set; }
+        public WindmillStatus Status { get; private set; }
+
+
+        [SerializeField] private GameObjectEvent onCityObjectVisualized;
```

Comment:
deze heeft naast CityObject ook nog een dependency op het EventContainer event systeem. dat zouden we onafhankelijk maken toch?